kolibri_instant_schools_plugin/assets/src/views/select-profile-page/profiles-list.vue

```diff
@@ -1,34 +1,47 @@
 <template>
 
-  <div>
-    <div
-      v-for="profile in profiles"
-      class="profile-card"
-      :class="{ 'profile-card-disabled': disabled }"
-      :key="profile.username"
-      role="button"
-      @click="selectProfile(profile)"
-    >
-      <div class="profile-icon">
-        <mat-svg
-          category="social"
-          name="person"
-          class="person-icon"
+  <table class="table">
+    <tr v-for="profile in profiles" :key="profile.username">
```

Comment:
I think these rows should have some sort of background-color/border.

Reply:
I'm not sure what this would look like? Let's make a new issue for this since there was already a bunch of discussion